packages/puppeteer-core/src/bidi/Connection.ts

```diff
@@ -141,9 +141,11 @@ export class BidiConnection
           return;
         case 'event':
           if (isCdpEvent(object)) {
+            // Remove `goog:` prefix from BiDi+ events.
+            const eventName = object.params.event.replaceAll('goog:', '');
```

Comment:
I don't think this line is needed. The event should be the raw CDP name.
The prefixed with `goog:cdp.` is on the method object.

Reply:
good point! Removed